selfdrive/car/chrysler/interface.py

```diff
@@ -220,8 +179,13 @@ def update(self, c, can_strings):
     self.gas_pressed_prev = ret.gasPressed
     self.brake_pressed_prev = ret.brakePressed
     self.cruise_enabled_prev = ret.cruiseState.enabled
+    self.left_blinker_prev = ret.leftBlinker
+    self.right_blinker_prev = ret.rightBlinker
+
+    # copy back carState packet to CS
+    self.CS.out = ret.as_reader()
 
-    return ret.as_reader()
+    return copy.copy(self.CS.out)
```

Comment:
Why do we need this copy?

Reply:
`CS.out` is used by carcontroller and wanted to avoid unexpected mods that might happen if the output of this method is modified.

Reply:
Can you even make mods to a capnp reader (you call as_reader above it)?

Reply:
yeah, you can't. I thought the as_reader was just to get rid of the traversal limit. Ok, I'll remove the copy.